src/styles/main.css

```diff
@@ -271,29 +271,69 @@
     /* Membership Markdown Content */
 
     .membership-content {
+        h1 {
+            margin-top: 2rem;
+            margin-bottom: 1rem;
+            font-size: 2rem;
+            color: #5569B0;
+            
```

Comment:
Indentation needs to be fixed if it is supposed to be nested within `membership-content`